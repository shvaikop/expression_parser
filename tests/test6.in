5
abc= 15.567 * 4 - 3
BB= 21 / 4 + $ABc / 6
bb= 31.5 % 2
cc= 123 + 5 * -123 -     5
dd= 60    * 4 % $cc + 21.5/ 1.5 
