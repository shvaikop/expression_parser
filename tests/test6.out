59.268
14.878
invalid
-497
254.333
